notary-eth-integration-test/src/integration-test/kotlin/integration/eth/WalletWithdrawalPipelineIntegrationTest.kt

```diff
@@ -229,6 +240,15 @@ class WalletWithdrawalPipelineIntegrationTest {
                 masterBalanceInitial.subtract(bigIntegerValue),
                 masterBalanceActual
             )
+
+            Thread.sleep(15_000)
```

Comment:
I hope the removal of obsolete modules does not take too much time

Reply:
Yes it should be done soon :)